### PR TITLE
Project structure

### PR DESCRIPTION
**Pull request by guilhermevrs**
Tib,
Take a look and see what do you think. Try to think about something that will possibly grow a lot

Tks

### CODE REVIEW

File: src/app/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#App main folder
+
+Remember to separate each component in its own folder
+
+/app
+   /component1
+       component1.ts
+       component1.html
```
Comment (tib-tib) [inline src/app/README.md]: component1.jade ;)

**Comment by tib-tib** [comment]
@guilhermevrs I think we should also add a app/services directory in order to organize the services we'll need (like for instance a service that enables us to interact with Facebook Graph API)

**Comment by guilhermevrs** [comment]
Agreed!
I would say even more, I think we need to step back and draw a simple architecture scheme to understand how we are going to develop all this stuff.
Give me some time to run through the angular 2 docs, so I get you back on this

**Comment by guilhermevrs** [comment]
I was reading up and I believe we could have, instead of /app/bindings, the following:

/app/shared - Shared components used through the app
/app/services - Service classes

What do you think @tib-tib ?

**Comment by tib-tib** [comment]
Fine by me @guilhermevrs

**Comment by guilhermevrs** [comment]
Done

**Comment by tib-tib** [comment]
selfie-0

**Comment by tib-tib** [comment]
@guilhermevrs btw you should install chrome extension github selfie ;)

**Comment by guilhermevrs** [comment]
hahaha, bg!
Can you please merge and close?

**Comment by tib-tib** [comment]
@guilhermevrs done

**Pull request merged**